setup.py

```diff
@@ -82,6 +82,7 @@
 
 REQUIRES = [
     'Babel==2.5.3',
+    'bleach==3.1.5',
```

Comment:
I think this is a pretty solid tool library to have in reqs https://github.com/mozilla/bleach